src/components/TodoList.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  Dispatch, SetStateAction,
+} from 'react';
+import { TodoItem } from './TodoItem';
+import { Todo } from '../types/Todo';
+import { ErrorTypes } from '../types/ErrorTypes';
+
+type Props = {
+  todos: Todo[],
+  setTodos: Dispatch<SetStateAction<Todo[]>>,
+  tempTodo: Todo | null,
+  setError: Dispatch<SetStateAction<ErrorTypes | null>>,
+  inputRefAdd: HTMLInputElement | null
+};
+
+export const TodoList:React.FC<Props> = ({
+  todos, setTodos, tempTodo, setError, inputRefAdd,
+}) => {
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings